src/app/component/comments/components/comment-body/comment-body.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="comment-body-wrapper">
+  <div class="comment-avatar">
+    <img src="../../../../../assets/img/comment_author.png" alt="comment-author">
+  </div>
+  <div class="comment-details">
+    <div class="comment-name-date">
+      <div class="author-name"><span>Jane Long Name Doe</span></div>
+      <div class="comment-date"><span class="comment-dot"></span>
+        <span class="comment-date-month">Jan 30, 2020</span>
+      </div>
+    </div>
+    <div class="comment-likes"><img src="../../../../../assets/img/comments/like.png" alt="like">
+      <span class="like-amount">999+</span>
+    </div>
+  </div>
+  <div class="comment-main-text">
+    <p class="comment-text">In this brilliantly conceived tour de force, Stephen King—who has absorbed the social,
+      political, and popular culture of his generation more imaginatively and thoroughly than
+      any other writer—takes readers on an incredible journey into the past and the possibility
+      of altering it.</p>
+  </div>
+  <app-comment-like-reply></app-comment-like-reply>
+</div>
```

Comment:
Please, add empty line at the end of file

Reply:
done already